src/neo-asm-cli/CommandLineOption.cs

```diff
@@ -0,0 +1,56 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace neo_asm_cli
+{
+    class CommandLineOption
+    {
+        public List<string> inputfiles;
+        public string outputfile;
+        public static CommandLineOption Parse(string[] args)
```

Comment:
Instead of rolling our own command line parser, can we just use an existing library? Neo-Express uses https://github.com/natemcmaster/CommandLineUtils/